example.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+// helloWorld is a function that does stuff 
```

Comment:
```suggestion
// helloWorld {{ funcComment }} 
```

This is a function, but you probably already knew that.

Edit the suggestion to use an alternative funcName.